Content.Shared/GameObjects/Components/SharedStackComponent.cs

```diff
@@ -57,6 +65,7 @@ public override void ExposeData(ObjectSerializer serializer)
         {
             serializer.DataFieldCached(ref _maxCount, "max", 50);
             serializer.DataFieldCached(ref _count, "count", MaxCount);
+            serializer.DataFieldCached(ref _spritestates, "spritestates", SpriteStates);
```

Comment:
add a check if it actually got 3 spritestates and throw an exception if it did not

Reply:
also check if spritestates are null here and throw an exception if so

Reply:
Shouldn't there be the possibility for stack-items without different spritestates? I didn't find any for phoron and floortiles in ss13, but might have overlooked them.

Reply:
you are right, this isn't purely visual